package.json

```diff
@@ -48,7 +48,7 @@
     "gulp-tslint": "^5.0.0",
     "gulp-util": "^3.0.6",
     "gulp-watch": "^4.3.4",
-    "harp": "git://github.com/filipesilva/harp.git#8da8d3497ddbfcbcbadd8be63e0fd731d7310cc4",
+    "harp": "^0.21.0-pre.0",
```

Comment:
Yes, it is `-pre.1`.
> semver resolution falls back to alphabetical

Good to know. Though, I wouldn't expect more than 9 pre subreleases. You still want the caret removed?